lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx/unordered_map/TestDataFormatterLibccUnorderedMap.py

```diff
@@ -64,3 +64,131 @@ def test_iterator_formatters(self):
                 ValueCheck(name="second", summary='"Qux"'),
             ],
         )
+
+        lldbutil.continue_to_breakpoint(process, bkpt)
+
+        # Test references to std::unordered_map
+        self.expect_var_path(
```

Comment:
Given that this is just for testing reference resolution, I'd consider making all of the maps have the same content (or even be references to the same object) so that this can be checked with a for loop. (The type will be different, but maybe we don't have to check that since that doesn't come from the data formatter?)

Reply:
makes sense. Simplified this in the latest commit. I just passed the "type" as a parameter to a helper function